packages/browser/src/index.js

```diff
@@ -24,14 +24,12 @@ import * as constants from './constants';
 // shared theme
 import colors from './components/theme/colors';
 
-import './antd.css';
-
 // shared store
 const store = configureStore();
 
 function WithConfigProvider(props) {
 	return (
-		<ConfigProvider prefixCls="dejavu-browser">
```

Comment:
@SavvyShah Why did you remove `prefixCls` prop?

Reply:
This is why we were having a [separate stylesheet before](https://github.com/appbaseio/dejavu/pull/451/files#diff-6b5ccd864a497944e16cd0dab1a78d4c936b974d4bca3c1168830cd700f18bb7). This doesn't allow the user of package to style the components.